auth/src/main/java/com/firebase/ui/auth/util/signincontainer/SmartLockBase.java

```diff
@@ -132,10 +134,6 @@ public void onStop() {
     @Override
     public void onDestroy() {
         super.onDestroy();
-        cleanup();
-    }
-
-    public void cleanup() {
```

Comment:
@samtstern What size level of changes do you want me to split into other PRs? I'm happy to move this to another PR. (it was just a small nit Intellij complained about)